scripts/check-torchcuda.py

```diff
@@ -60,20 +60,21 @@ def err(s: str) -> None:
     if not file.endswith(".cpp"):
```

Comment:
@eternalNight what do you think about explicitly checking for python file modifications?

```suggestion
    if file.endswith(".py"):
```